LocalSettings.php

```diff
@@ -4876,10 +4877,10 @@
 		'default' => false,
 	],
 	'wgWikiSeoEnableAutoDescription' => [
-		'default' => false,
+		'default' => true,
 	],
 	'wgWikiSeoTryCleanAutoDescription' => [
-		'default' => false,
+		'default' => true,
```

Comment:
overridedefault needs changed in ManageWikiSettings.php.

Reply:
Changed, thanks.